scripts/create-latest

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+
+import os
+from subprocess import run, DEVNULL, PIPE
+
+# We *probably* won't change any of these, but at least it means we
+# could if we wanted to.
+LATEST = "latest"
+ORIGIN = "origin"
+MASTER = "master"
+UPDATE = "./src/update_versions_to_latest_master.py"
+
+
+def subprocess_output(x):
+    return x.stdout.decode("utf-8").rstrip().split("\n")
+
+
+def submodule_status():
+    res = run(["git", "submodule", "status"], stdout=PIPE, check=True)
+    lines = subprocess_output(res)
+    parsed = [[x[0] != " "] + x[1:].strip().split(" ")[:2] for x in lines]
```

Comment:
This is because you get a blank space at the start of the submodule status line if it matches the repository and some  character like + or - if it doesn't, so you can tell if local has moved on? Some light commenting for dullards like me might be nice here!